awesome_print.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rspec',  '>= 3.0.0'
   s.add_development_dependency 'appraisal'
   s.add_development_dependency 'fakefs', '>= 0.2.1'
-  s.add_development_dependency 'sqlite3'
+  s.add_development_dependency 'sqlite3', '~> 1.3', '< 1.4'
```

Comment:
I will be happy to remove this change. It may be specific to my setup.

I've only managed to successfully run `rake` after adding that ☝️, without it, I was getting:

```
Failure/Error: ActiveRecord::Base.establish_connection adapter: 'sqlite3', database: ':memory:'

Gem::LoadError:
  Specified 'sqlite3' for database adapter, but the gem is not loaded. Add `gem 'sqlite3'` to your Gemfile (and ensure its version is at the minimum required by ActiveRecord).
# ./spec/active_record_helper.rb:10:in `<top (required)>'
# ./spec/ext/active_record_spec.rb:2:in `require'
# ./spec/ext/active_record_spec.rb:2:in `<top (required)>'
# ------------------
# --- Caused by: ---
# Gem::LoadError:
#   can't activate sqlite3 (~> 1.3.6), already activated sqlite3-1.4.1. Make sure all dependencies are added to Gemfile.
#   ./spec/active_record_helper.rb:10:in `<top (required)>'
```

See https://stackoverflow.com/questions/54527277/cant-activate-sqlite3-1-3-6-already-activated-sqlite3-1-4-0